migrations/148_assessments__assessment_set_id__nullable.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE assessments ALTER COLUMN assessment_set_id DROP NOT NULL;
+-- This will verify that any non-deleted assessments have a valid assessment set ID
+-- NOTE: This is potentially very expensive, as all rows will be checked.
+-- Keep this in mind when deploying.
+ALTER TABLE assessments ADD CONSTRAINT assessment_set_id_check CHECK (
+  deleted_at IS NOT NULL
+  OR (deleted_at IS NULL AND assessment_set_id IS NOT NULL)
+);
```

Comment:
Just looking ahead to minimizing downtime during deployment, I’m wondering if we should change this SQL to only add this constraint if it doesn’t already exist (may need plpgsql code?) and run the ALTER TABLE ahead of time by hand?